src/controllers/Projects.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * @fileOverview Projectd Controller
+ *
+ * @author Seun Agbeye
+ *
+ * @requires ../helpers
+ * @requires ../models
+ */
+
+import config from '../config';
+import helpers from '../helpers';
+import models from '../models';
+
+const { Op } = models.Sequelize;
+
+/**
+* Controls endpoints for team projects
+* @class Projects
+*/
+export default class Projects {
+  /**
+   * @method create
+   * @desc This method creates a new project
+   * for the team with the specified team ID
+   *
+   * @param { object} req request
+   * @param { object} res response
+   *
+   * @returns { object } response
+   */
+  async create(req, res) {
+    const { teamId } = req.params;
+    try {
+      const project = await models.Project.create({
+        ...req.body,
+        teamId
+      });
+      return res.sendSuccess({ project }, 201);
+    } catch (error) {
+      return res.sendFailure([error.message]);
+    }
+  }
+
+  /**
+     * @method deleteById
+     * @desc This method deletes the membership with the specified member ID
+     * from the team with the specified team ID
+     *
+     * @param { object} req request
+     * @param { object} res response
+     *
+     * @returns { object } response
+     */
+  async deleteProjectById(req, res) {
+    const { teamId, projectId } = req.params;
+    try {
+      const project = await models.Project.getOr404(projectId);
+      // await models.Project.validateTeam(project, teamId);
+      await project.destroy();
+
+      return res.status(200).send({
+        data: {},
+        message: 'Project successfully deleted'
+      });
```

Comment:
Ok.

Reply:
@Chieze-Franklin How do I pass the `message`  if I use `res.sendSuccess` , because it seems the function does not support sending `message` at the moment.

Reply:
Yes because many times messages are redundant. They contain information that is already known.

Reply:
I think that is deliberate because most times the info in the `message` is redundant. The consumer of the API can always know if the request succeeded or failed without any need for `message`.